docs/tutorial.md

```diff
@@ -4,10 +4,23 @@ title: Step-by-Step Tutorial
 description: This document provides a step-by-step tutorial for the reSolve framework. Throughout this tutorial, you will iteratively develop a ShoppingList application and learn fundamental concepts of the reSolve framework.
```

Comment:
`This document provides a step-by-step tutorial for the reSolve framework.`
Please rewrite without using "provides".

Reply:
I removed the first sentence completely. This description is for search engines.